### ISSUE TITLE
App móvil

### ISSUE THREAD

**Issue by xfrcc** [xfrcc/iptv-auto-activator#2]
Hay que pensar en una forma de traer actualizaciones en la lógica de la activación sin que tengan que descargar la App otra vez (en caso de que cambien algo como acaba de pasar con Kraken TV)

**Comment by IAMDROPPE** [comment]
No entiendo la cuestión del todo, pero usa solución fácil ante actualizaciones sin que el usuario tenga que descargar siempre el apk, es una app web la cual implementa lenguajes de programación utilizados para desarrollo web. También una app nativa con extensión a servidores web. El punto es que no se requiera actualizar a menos que debas hacer cambios en el núcleo del código de la app por así decirlo. No sé si me explico bien.

**Comment by IAMDROPPE** [comment]
Voy a invitar a unos colegas a ver si pueden meterse en el tema.

**Comment by pcurz** [comment]
Hola, estoy haciendo la app cuando esté funcional cambiará el repo

**Comment by pcurz** [comment]
[app-debug.apk.zip](https://github.com/pcurz/iptv-auto-activator/files/10752961/app-debug.apk.zip)
aqui hay una preview de la app, funciona la activación, pero la interfaz está pobre y falta hacerlo modular cosa de que puedan agregar más servicios, cuando eso lo tenga listo lo subiré al repo, pero mientras esta apk sirve para ir activando (ojo renombrar el archivo, quitar el .zip del nombre eso era porque no me permitia subir .apk github)

**Comment by IAMDROPPE** [comment]
> [app-debug.apk.zip ]

Está excelente la app, lástima que aún no funciona con kraken, sin embargo, ahora es más fácil activarla, solo debes crearte una cuenta a través de una apk para añadir hasta 3 dispositivos y activarla en un enlace web muy similar como era antes, solo que esta vez termina con una key, intente hacer la activación modificando tu código en nodejs, pero no lo logre, aún no tengo tu nivel; sin embargo, cree un script muy básico para activar.

Por aquí es algo complicado conversar, si me facilitas un medio más efectivo como Telegram o un email podría explicarte un poco más de forma detallada.

**Comment by matichewer** [comment]
> > [app-debug.apk.zip ]
> 
> Está excelente la app, lástima que aún no funciona con kraken, sin embargo, ahora es más fácil activarla, solo debes crearte una cuenta a través de una apk para añadir hasta 3 dispositivos y activarla en un enlace web muy similar como era antes, solo que esta vez termina con una key, intente hacer la activación modificando tu código en nodejs, pero no lo logre, aún no tengo tu nivel; sin embargo, cree un script muy básico para activar.
> 
> Por aquí es algo complicado conversar, si me facilitas un medio más efectivo como Telegram o un email podría explicarte un poco más de forma detallada.

El script que hiciste es para activar KrakenTv ? No lo encontré en tu perfil.
Yo me quedé en el tiempo con el activador de SplikTv en éste [repo](https://github.com/matichewer/Activate-SplikTv-app), una lástima que desapareció esa app

**Comment by IAMDROPPE** [comment]
> > > [app-debug.apk.zip ]

No, solo activa teleclub, quizá más adelante agregue la opción de activar kraken pero por ahora no se activa desde la app ni desde el script. hay que hacer varios cambios para que funcione.

**Comment by IAMDROPPE** [comment]
> > > [app-debug.apk.zip ]
> > 
> > 
> > Está excelente la app, lástima que aún no funciona con kraken, sin embargo, ahora es más fácil activarla, solo debes crearte una cuenta a través de una apk para añadir hasta 3 dispositivos y activarla en un enlace web muy similar como era antes, solo que esta vez termina con una key, intente hacer la activación modificando tu código en nodejs, pero no lo logre, aún no tengo tu nivel; sin embargo, cree un script muy básico para activar.
> > Por aquí es algo complicado conversar, si me facilitas un medio más efectivo como Telegram o un email podría explicarte un poco más de forma detallada.
> 
> El script que hiciste es para activar KrakenTv ? No lo encontré en tu perfil. Yo me quedé en el tiempo con el activador de SplikTv en éste [repo](https://github.com/matichewer/Activate-SplikTv-app), una lástima que desapareció esa app

No había entendido bien tu. Mensaje,
Si, tengo un script personal, pero uso un método distinto en python. Ya que se necesita estar registrado en una App llamada zona TV Box. De lo contrario es imposible o mejor dicho, imposible para mi activarla de otra forma.

**Comment by matichewer** [comment]
Claro, yo actualmente estoy activando KrakenTV manualmente con ZonaTVbox. Es un embole

**Comment by IAMDROPPE** [comment]
> Claro, yo actualmente estoy activando KrakenTV manualmente con ZonaTVbox. Es un embole

Yo tuve que copiarme del método de **pcurz** pero en Python, porque no le se mucho al js , sin embargo, sigue siendo un embole porque me falla ya que ahora es por key en lugar de token y si se tiene mas de un dispositivo empeora todo xd

**Comment by pcurz** [comment]
> > [app-debug.apk.zip ]
> 
> Está excelente la app, lástima que aún no funciona con kraken, sin embargo, ahora es más fácil activarla, solo debes crearte una cuenta a través de una apk para añadir hasta 3 dispositivos y activarla en un enlace web muy similar como era antes, solo que esta vez termina con una key, intente hacer la activación modificando tu código en nodejs, pero no lo logre, aún no tengo tu nivel; sin embargo, cree un script muy básico para activar.
> 
> Por aquí es algo complicado conversar, si me facilitas un medio más efectivo como Telegram o un email podría explicarte un poco más de forma detallada.

tengo telegram es @pcurz, por alguna razón no me esta notificando por correo acá por eso no respondía jaja, cuando subí la apk si estaba activando kraken tv ahora recien me entero que cambiaron todo

**Comment by pcurz** [comment]
> Claro, yo actualmente estoy activando KrakenTV manualmente con ZonaTVbox. Es un embole

Se me ocurre con un emulador de android usar http toolkit para capturar las request de la app, cosa de hacerlo todo en uno, el registro + el token que te asigne para usarlo para la activacion

**Comment by IAMDROPPE** [comment]
> > Claro, yo actualmente estoy activando KrakenTV manualmente con ZonaTVbox. Es un embole
> 
> Se me ocurre con un emulador de android usar http toolkit para capturar las request de la app, cosa de hacerlo todo en uno, el registro + el token que te asigne para usarlo para la activacion De poder se puede claro que cada vez comienza a extenderse la forma en la que obtenemos el cómo funciona, antes era literalmente un get y un post de una misma pagina

Ya te escribí por telegram planteando otro método, por así decirlo.
Por aquí, también se me hace complicado responder,

**Comment by pcurz** [comment]
No me deja iniciar sesión telegram, le mandé un correo para que me lo solucionen

**Comment by matichewer** [comment]
Alguien tiene un script que active los dispositivos? Al parecer es por token de tu cuenta de ZonaTvBox, y solo puedes registrar hasta 3 dispositivos por cuenta

**Comment by pcurz** [comment]
> Alguien tiene un script que active los dispositivos? Al parecer es por token de tu cuenta de ZonaTvBox, y solo puedes registrar hasta 3 dispositivos por cuenta

En teoría como está actualmente es mas conveniente porque podrás activar independiente si estás en la misma red, solo con el token

**Comment by IAMDROPPE** [comment]
Que te mejores bro, más bien haces mucho.
De verdad te lo agradezco.
